src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -41,14 +51,15 @@ const NavHeader = (props) => {
     onCancelToken,
     placeholderText,
     setIsFirstTimeOpen,
-    getSearchLayout
+    getSearchLayout,
+    opacityImage
   } = props;
   const navigation = useNavigation();
   const coverPath = topicDetail?.cover_path || null;
   const [domainHeight, setDomainHeight] = React.useState(0);
   const [searchHeight, setSearchHeight] = React.useState(0);
-  const {top} = useSafeAreaInsets();
-  const topPosition = Platform.OS === 'ios' ? top : 0;
+  const insets = useSafeAreaInsets();
+  const {width: displayWidth} = useWindowDimensions();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of `useWindowDimensions` is a good addition to get the window dimensions in a more efficient way. However, it's important to note that this hook causes re-renders when the window's dimensions change (e.g., device rotation). If this behavior is not desired, consider using `Dimensions.get('window')` instead.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it's desired,